lib/amazing_print/formatters/base_formatter.rb

```diff
@@ -123,7 +123,6 @@ def indent(n = indentation)
       end
 
       def outdent
-        ' ' * (indentation - options[:indent].abs)
```

Comment:
@paddor it looks like this wasn't doing anything, right?

Reply:
Yeah, looks like it.